mbofui/app/js/app.js

```diff
@@ -17,6 +17,17 @@ ocellus.run(function($rootScope) {
       'modal_endtime':'End time',
       'modal_post':'Add Event'
     };
+    $rootScope.event_categories = [
```

Comment:
Is this assignment a temporary implementation until the REST API provides access to categories from the DB?